Assets/Scripts/DDAA/Variables/EnemySpawnDDAA.cs

```diff
@@ -34,8 +34,8 @@ public static EnemySpawnDDAA Instance
     // All of the parameters below are the ones to change, when adjusting the DDA (unless there's a bug)
 
     //static parameters
-    private static readonly float baseSpawnPoint = 4f;
-    public static readonly int minSpawnAmount = 7;
+    private static readonly float baseSpawnPoint = 2f;
+    public static readonly int minSpawnAmount = 3;
```

Comment:
So you essentially reduced the amount of enemies spawning in total? We talked about increasing the gameplay length right?

Reply:
I saw we affect the spawn amount mostly with static addition. We have to discuss that

Reply:
We'll fix it later